cmd/srcd/cmd/parse.go

```diff
@@ -133,18 +154,13 @@ var parseLangCmd = &cobra.Command{
 		if err != nil {
 			return fmt.Errorf("could not get daemon client: %v", err)
 		}
-		ctx, cancel := context.WithTimeout(context.Background(), time.Second)
-		defer cancel()
 
-		res, err := c.Parse(ctx, &api.ParseRequest{
-			Kind:    api.ParseRequest_LANG,
-			Name:    path,
-			Content: b,
-		})
+		lang, err := parseLang(context.Background(), c, path, b)
 		if err != nil {
-			return fmt.Errorf("server error: %v", err)
+			return fmt.Errorf("%v", err)
```

Comment:
For consistency with [`parseUASTCmd.RunE`](https://github.com/src-d/engine/pull/186/files#diff-9250d38c157246923d9a07a480ccb428R87) it should be:
```suggestion
			return fmt.Errorf("cannot parse language: %v", err)
```